shell/common/gin_converters/guid_converter.h

```diff
@@ -7,13 +7,17 @@
 
 #if defined(OS_WIN)
 #include <rpc.h>
+
+#include "base/strings/sys_string_conversions.h"
+#include "base/win/win_util.h"
 #endif
 #include <string>
 
 #include "gin/converter.h"
 
 #if defined(OS_WIN)
 typedef GUID UUID;
+const GUID GUID_NULL = {0, 0, 0, {0, 0, 0, 0, 0, 0, 0, 0}};
```

Comment:
I think this will work fine and auto-initialize to 0: https://en.cppreference.com/w/cpp/language/zero_initialization
```suggestion
const GUID GUID_NULL;
```

Reply:
This behavior is behind [an option](https://reviews.llvm.org/D54604) and it seems that Chromium does not use auto zero initialization:
https://source.chromium.org/chromium/chromium/src/+/master:build/config/compiler/BUILD.gn;l=2514?ss=chromium

It uses pattern initialization to fill integers with 0xAA automatically so uninitialized variables are more liked to cause crashes.

Reply:
@nornagon So are we ok to leave the explicit initialization?

Reply:
Will an explicitly-initialized but empty initializer list work?
```suggestion
const GUID GUID_NULL = {};
```

Reply:
Yep it works. Changed!